docs/technical-reference/contributing.md

```diff
@@ -110,3 +110,14 @@ Once you have made sure your changes work as expected, you are ready to submit t
 
 Your changes will be reviewed and we might suggest improvements to your pull request. You can submit those follow-up changes by adding more commits to your branch. Your commits will generally be squashed when merging the pull request.
 Thank you for your contribution!
+
+## Contributing as a designer
+
+There are plenty of areas where we would welcome help from design-oriented contributors, be it to improve existing features or to develop new ones.
+If you want to give us a hand in this capacity, we recommend that you first get acquainted with OpenRefine as a user.
+[Install OpenRefine](docs/manual/installing.md) and learn to use it by following [some tutorials](/external_resources) or watching [some videos](https://www.youtube.com/watch?v=B70J_H_zAWM&list=PLYMbXPIhZRlVgxgljjZu6rsFl1yeXXLvp). Even better, you could
+use the tool on a dataset you are familiar with, to transform it and link it to other data sources, for instance.
+
+Perhaps in that process you already encounter things that are counter-intuitive for you, workflows that could be improved, interfaces that could be made clearer, and so on. if that is the case, it is very useful if you give us this feedback. It could be on the forum, for instance in the Support category if you are not sure about something, or in the Design proposals category if you have a proposal for an alternative design. You could file a GitHub Issue directly.
```

Comment:
```suggestion
Perhaps in that process you already encounter things that are counter-intuitive for you, workflows that could be improved, interfaces that could be made clearer, and so on. If that is the case, it is very useful if you give us this feedback. It could be on the forum, for instance in the Support category if you are not sure about something, or in the Design proposals category if you have a proposal for an alternative design. You could file a GitHub Issue directly.
```

Reply:
Everything else is good enough! Thanks!